src/js/components/Form/stories/ValidateOnChange.js

```diff
@@ -61,6 +65,7 @@ export const ValidateOnChange = () => {
             <Select
               name="select-size"
               id="select-size"
+              aria-label="select-size"
```

Comment:
Can we still add `htmlFor` and `id` in addition to `aria-label`?

Reply:
```suggestion
              htmlFor="select-size"
              id="select-size"
              aria-label="select-size"
```
I tested these changes locally and no issues are created by adding the htmlFor and id.